style.css

```diff
@@ -164,19 +164,19 @@ nav li a:hover{
 
 .left a{
     display:inline-block;
-    background-color:#CCB4E7;
-    color:white;
+    background-color:#ceeddb;

```

Comment:
Version 2's colour scheme consists of light greens complemented by dark for borders and hover functions.